src/app/Recordings/Recordings.tsx

```diff
@@ -61,7 +61,7 @@ export const Recordings: React.FunctionComponent<RecordingsProps> = (_) => {
 
   const cardBody = React.useMemo(() => {
     return archiveEnabled ? (
-      <Tabs id="recordings" activeKey={activeTab} onSelect={onTabSelect}>
+      <Tabs id="recordings" activeKey={activeTab} onSelect={onTabSelect} unmountOnExit>
```

Comment:
What does this `unmountOnExit` do?

Reply:
This means we only build the active tab and remove any inactive out of DOM. This helps in avoiding "jump" while rendering when depending on the toolbar size instead of the entire viewport.

Without this option, you will originally see the Kebab first then the buttons on large viewport.

Reply:
Kinda unfortunate but I think it has an advantage that first render should be faster as it only needs to build one table.

Reply:
https://www.patternfly.org/v4/components/tabs#tabs

> Unmounts tab children (removes them from the DOM) when they are no longer visible

Seems to make sense.